config/locales/money.ru.yml

```diff
@@ -0,0 +1,4 @@
+ru:
+  errors:
+    messages:
+      invalid_currency: должнa быть корректной валютой (например '100', '5%{decimal}24', или '123%{thousands}456%{decimal}78'). Получено %{currency}
```

Comment:
@semmons99 should we fix the logic here first? The message hints at an invalid currency, but it's actually suggesting how to format the number correctly

Reply:
good call. @kirillshamardin could you work with @antstorm on fixing the error definition

Reply:
@antstorm please check it now.